.circleci/config.yml

```diff
@@ -83,7 +91,16 @@ jobs:
           command: poetry check
           name: dependencies
       - run:
-          command: poetry run flake8 --ignore=E501,E402 .
+          command: |
+            sudo apt-get install jq
+            if COMMAND_OUTPUT=$(poetry check && poetry run flake8 --ignore=E501,E402 . 2>&1); then
```

Comment:
I can't tell if this if statement is checking something about COMMAND_OUTPUT (that is, if it's like C where an assignment returns the value stored) or if it's checking the return value of the poetry command...

I think it would be better to move the assignment outside the if statement; and have the if statement either check COMMAND_OUTPUT, or store the return value with something like COMMAND_RETURN=$? and then check the return value.